main/settings.py

```diff
@@ -29,6 +29,7 @@
     SECRET_KEY = os.environ["SECRET_KEY"]
```

Comment:
Provide a default value here:-
```suggestion
    SECRET_KEY = os.environ.get("SECRET_KEY", "default_value")
```